### ISSUE TITLE
The slice band cuts at 31 mm, and the two worst labels live at the edge

### ISSUE THREAD

**Issue by dk2lone** [dk2lone/knee#36]
`SLICE_BAND = (0.20, 0.80)` has never been challenged. It samples the middle 60% of each
ordered stack, and on a sagittal series that axis is medial-lateral, so the constant
decides how far toward each compartment a slice can be drawn from.

## What the headers say

`eda/band_extent.py`, from `n_slices` and the slice spacing:

| plane | series | median extent | band spans | half-width | band cut inside 30 mm |
|---|---:|---:|---:|---:|---:|
| Axial | 5,898 | 124.8 mm | 74.9 mm | 37.4 mm | 17.0% |
| Coronal | 8,609 | 105.0 mm | 63.0 mm | 31.5 mm | 40.4% |
| Sagittal | 9,864 | 102.3 mm | 61.4 mm | **30.7 mm** | **42.7%** |

A sagittal series is a median 30 slices at 3.3 mm. The band reaches 31 mm either side of
the stack centre, and the femoral condyles span roughly 70-80 mm. In 43% of sagittal
series the cut falls inside 30 mm.

## Why that is the suspect

The two worst labels on the public frontier are the two that live out at that edge (#35):

| label | public 20 | RadImageNet arm | teacher |
|---|---:|---:|---:|
| Lateral Meniscus | 0.660 | **0.722** | 0.879 |
| Lateral OA | 0.706 | **0.812** | 0.833 |

The arm that beats every DINOv2 member on both reads **(0.12, 0.88)**, not (0.20, 0.80).
It also differs in encoder, resolution and crop, so this is a suspicion and not a finding
- but it is the only one of those differences that is free to test and free to deploy.

Resolution is not the alternative explanation. 130 mm over 336 px is 0.387 mm/px, already
finer than the acquisition, so a tighter crop upsamples rather than resolves.

## What it costs to widen

Coverage trades against through-plane density. Twelve slices over 61 mm is 5.1 mm apart;
twelve over 98 mm is 8.2 mm apart. So a wider band is not free, and the intermediate rung
is in the sweep for exactly that reason.

## The sweep

```
.venv/bin/python -m modal deploy cloud/train.py
.venv/bin/python cloud/launch.py bands small 8 4
```

Three arms, one fold, eight epochs, 12 slices: (0.20, 0.80) as the control, then
(0.10, 0.90) and (0.02, 0.98). Each arm decodes its own pixels - the band changes what a
pixel is, and `eda/test_sweep_band.py` fails if the memo ever lets one arm reuse another's
cache. That costs about half an hour more per arm than an adaptation sweep.

Read `oof` and `focal`; `va_sel` and `gold` come out NaN at one fold (#33). If a wider
band lifts the two lateral labels it lifts them for every member and costs nothing at
inference - only the members have to be refitted under it.

Blocked until the parity run frees the app: a deploy against `knee-train` while it is
training would risk the run.

**Comment by dk2lone** [comment]
Independent corroboration for the shape of this, from stevenleehans on the forum
(discussion/735154, 17 h ago), whose team runs the same 6-slot pipeline at 224 px:

- DINOv2-small → base moved their CV by **+0.0011 against a noise floor of 0.0020**, and
  RAD-DINO lost outright. That is our sweep's answer arrived at independently (#33), so
  the encoder axis is closed from two directions.
- **A crop-geometry fix earlier in their log paid +0.0059 and moved 10 of 12 labels.**
  They contrast that with the encoder null, which moved 5 of 12 and rested entirely on
  MCL. Geometry is where their gains came from; capacity was not binding.
- Their weakest labels on 2,652 weak-labelled studies are Lateral Meniscus 0.7385 and
  Lateral OA 0.7603 — the same two, on a different label set and a different fold scheme.

Three independent measurements now say the same thing about which labels are unsolved,
and the one lever measured to move ten labels at once is geometry, not capacity. The band
is a geometry change.

**Comment by dk2lone** [comment]
## Tested offline before spending the sweep, and it does not hold

The band is a *fraction* of each stack, so how many millimetres it covers varies by study.
If cutting the compartment were the mechanism, our narrow-band model should be worse on
the studies where the band covers the fewest millimetres. It is not.

Diff-in-difference over all 4,407 studies, our five-fold train-v1 OOF against the arm's
published OOF, weak labels as truth, split by the band's absolute half-width:

| band half-width | n | Lateral Meniscus, ours | rad | rad − ours |
|---|---:|---:|---:|---:|
| 18-30 mm | 1,776 | 0.687 | 0.798 | +0.112 |
| 30-32 mm | 1,169 | 0.684 | 0.831 | +0.147 |
| 32-49 mm | 1,462 | 0.700 | 0.772 | +0.072 |

Our model scores 0.687 where the band reaches 18-30 mm and 0.700 where it reaches
32-49 mm. That is +0.013 across the whole range of the thing this issue proposed as the
cause, and the arm's advantage is not larger where our band is narrower - it is smallest
there. On Lateral OA our own score moves the wrong way entirely (0.765 narrow, 0.691
wide).

The crop fails the same test. 99.8% of coronal series have a field of view wider than the
130 mm crop, so the crop always discards periphery, and splitting by how much it discards:

| coronal FOV | n | Lateral Meniscus, rad − ours | Lateral OA | PF OA |
|---|---:|---:|---:|---:|
| small | 2,127 | +0.117 | +0.090 | +0.106 |
| mid | 900 | +0.080 | +0.130 | +0.166 |
| large | 1,380 | +0.106 | +0.027 | +0.054 |

The gap is smallest where our crop throws away the most. Both geometry suspects predict a
gradient and neither produces one.

**So the arm's advantage on the two lateral labels is not the pixels it samples.** What is
left is what it is: RadImageNet pretraining, and a head that attends over every slice
token with one query per finding, against our per-slot pooling. Those are confounded with
each other and neither is free to test.

Closing this.  and the per-arm / plumbing stay - the
sweep is one command if a reason to run it appears - but it is not scheduled, and the
4 GPU-hours go to the parity run instead.

Caveat, stated because it is the one that could reverse this: train-v1 reads six slots and
only three of them are sagittal, so the coronal and axial slots may carry the lateral
compartment whatever the sagittal band does. That would make our model insensitive to the
band without making the band harmless for a sagittal-only reader.

**Comment by dk2lone** [comment]
Correction to the line above that the shell ate: the sentence should read —

Closing this. `eda/band_extent.py` and the per-arm `band` / `crop_mm` plumbing stay, so
the sweep is one command if a reason to run it appears, but it is not scheduled and the
4 GPU-hours go to the parity run instead.
